2-Webpack/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+    mode: 'development',
+    entry: {
+        main: path.resolve(__dirname, 'scripts/todo_list.js')
+    },
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: '[name].[contenthash].js',
+        clean: true
+    },
+    devtool: 'inline-source-map',
+    devServer: {
+        static: './dist',
+        port: 5001,
+        open: true,
+        hot: true
+    },
+    //loaders
+
+    //plugins
+    plugins: [new HtmlWebpackPlugin({
```

Comment:
seems like the actual capabilities of `HtmlWebpackPlugin` are not being used.. is there a reason it's here?

Reply:
It adds the index.html file with a link to the bundle.js file, doesn't it?

Reply:
if that's the case, remove the `link`s from your `index.html`

Reply:
The link to the JS file is removed. The ones you see are for the CSS files which aren't added by the copy plugin